src/Compilers/CSharp/Portable/CommandLine/CSharpCommandLineParser.cs

```diff
@@ -615,6 +616,17 @@ public new CSharpCommandLineArguments Parse(IEnumerable<string> args, string? ba
                             }
                             continue;
 
+                        case "generatedfilesout":
+                            if (RoslynString.IsNullOrWhiteSpace(value))
```

Comment:
>RoslynString [](start = 32, length = 12)

nit: I think `string.IsNullOrWhiteSpace` would do (`RoslynString` was there to add nullability annotations, but shouldn't be necessary now)